eclass/dune.eclass

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: dune.eclass
+# @MAINTAINER:
+# maintainer-needed@gentoo.org
```

Comment:
Add yourself as a maintainer, I don't think eclass needs to be maintained by "@gentoo.org" person.